intertidal/io.py

```diff
@@ -822,9 +822,7 @@ def _ls_platform_instrument(year):
 
 def prepare_for_export(
     ds,
-    int_bands=None,
-    int_nodata=255,
-    int_dtype=np.uint8,
+    custom_dtypes=None,
```

Comment:
Replaced setting all integer bands to the same dtype/nodata to a more flexible dictionary approach where each band can have its own dtype and nodata valuye